views/components/ScrollContainer.svelte

```diff
@@ -96,10 +126,20 @@
               Image n+1 is below and becomes visible
               => smooth transition n to n+1
       -->
-      {#each imageUrlsReverse as { id, image }}
+      {#each imageUrlsReverse as { id, image }, i}
         {#if image && [index - 1, index, index + 1].includes(id)}
+          <!-- 
+            Fade out animation is skipped if the appearing image has not the same aspect ratio.
+            Instead the appearing image is faded in.
+            Otherwise the fading image changes dimensions because it applies the updated 'imageHeight' of the appearing image during fade out.
+          -->
           <img
             class="q-scroll-graphic-image"
+            class:q-scroll-graphic-image--transition-animation={item.options
```

Comment:
I noticed that the `options` object is also not available on existing items (see screenshot and [link to article](https://www-stage.nzz.ch/international/klimamodelle-wie-sie-funktionieren-und-was-man-von-ihnen-erwarten-kann-ld.1337371)). I think it would make sense to define a new variable which checks if the options and disableAnimation are available.
<img width="1304" alt="Screenshot 2021-12-02 at 11 23 40" src="https://user-images.githubusercontent.com/1810384/144404262-c6b78dea-01dc-43f3-96fc-68cdada7092f.png">

Reply:
Thanks you're too fast! I didn't test it yet but should be fixed now. :)
The options property will now be initialised empty, inside ScrollGraphic.svelte, if not present.

Reply:
Ah sorry heheh